### ISSUE TITLE
 Cannot read property 'context' of undefined

### ISSUE THREAD

**Issue by duan602728596** [webpack-contrib/worker-loader#125]
When I use webpack (version: 4.0.0-beta.2) and worker-loader (version: next), it has an error:
```
ERROR in ./node_modules/worker-loader/dist/cjs.js?name=worker!./src/worker.js
Module build failed: TypeError: Cannot read property 'context' of undefined
    at Object.pitch (E:\gongcheng\qqtools\worker-loader\ceshi\node_modules\worker-loader\dist\index.js:71:45)
 @ ./src/index.js 1:0-56
```
I found the error here: [https://github.com/webpack-contrib/worker-loader/blob/next/src/index.js#L40](https://github.com/webpack-contrib/worker-loader/blob/next/src/index.js#L40)

I found that the code changed from
```javascript
const filename = loaderUtils.interpolateName(this, options.name || '[hash].worker.js', {
  context: options.context || this.options.context,
  regExp: options.regExp,
});
```
to
```javascript
const filename = loaderUtils.interpolateName(this, options.name || '[hash].worker.js', {
  context: this.context || options.context || this.options.context,
  regExp: options.regExp,
});
```
It is no problem.

**Comment by michael-ciniawsky** [comment]
`webpack` v4.0.0 changed `this.context` => `this.rootContext`

```js
context: options.context || this.rootContext || this.context
```

should work. PR welcome :)

**Comment by michael-ciniawsky** [comment]
Fixed by #128 , released in [`v1.1.1`](https://github.com/webpack-contrib/worker-loader/releases/tag/v1.1.1) 🎉
